_sass/_custom-components.scss

```diff
@@ -0,0 +1,40 @@
+// ==========================================================================
+// Custom non-USWDS Components - Break it out per component if file gets too long.
+// ==========================================================================
+
+// ==========================================================================
+// News List - Seen on homepage
+// ==========================================================================
+.news-list {
```

Comment:
11. These are the custom styles for the News and Updates component. It's using design tokens to set spacing. You can find out more about spacing units [here](// https://designsystem.digital.gov/design-tokens/spacing-units/)